packages/core/src/components/badge/badge.tsx

```diff
@@ -0,0 +1,23 @@
+import { Component, h, Prop } from '@stencil/core';
+import clsx from 'clsx';
+
+@Component({
+  tag: 'dso-badge',
+  styleUrl: 'badge.scss',
+  shadow: true
+})
+export class Badge {
+  @Prop() condition?: 'primary' | 'success' | 'info' | 'warning' | 'danger';
+  @Prop() status?: string;
```

Comment:
Ik ben zelf nog niet helemaal tevreden met deze prop-naam, maar zo heet hij in het huidige component ook. Goede alternatieven zijn welkom.

Reply:
Maak er maar weer `level` van, `condition` staat wat mij betreft te dicht bij "voorwaarde".

En net zoals Class Decorators zetten we de Property decorators op een eigen regel

Reply:
Ik vind dit er zo gek uit zien

```
export class Badge {
  @Prop()
    level?: 'primary' | 'success' | 'info' | 'warning' | 'danger';
  @Prop()
    status?: string;
```

Reply:
Zoals besproken, elke @Prop() een eigen regel, met witregels tussen de @Props en geen indenting op de decorator